github/src/main.js

```diff
@@ -274,6 +290,10 @@ function parseIssueRow(requestedFields, issue) {
         return issue.locked;
       case 'title':
         return issue.title;
+      case 'label':
```

Comment:
Checked to make sure this works, and looks like it's close, but not quite there. If you look at the [list issues] endpoint, you can see that issues have many labels, not just one. I think we would need make the schema have "labels" instead of "label" for the name, and we would need to map over the labels for the issue grab the `.name`, and then `.join(',')`. 

It also looks like we would want `issue.milestone && issue.milestone.title` since `milestone` is an object. 
[list issues]: https://developer.github.com/v3/issues/#list-issues